docs/installation/installation-allenact.md

```diff
@@ -104,10 +107,15 @@ pip install -r allenact_plugins/<PLUGIN_NAME>_plugin/extra_requirements.txt
 
 from the top-level directory.
 
-<!--
-#### Installing requirements with `conda`
 
-Clone the repository to your local machine and move into the top-level directory
+## Installing a `conda` environment
```

Comment:
Can you move this into the `### Full Library` heading and then change `## Installing requirements with pip` to something like `## Different ways to use allenact`. I think this will help make clear that the conda installation is mainly meant as an alternative to doing something like `pip install -r requirements.txt`.

Reply:
I did it, and also slightly restructured the page (including moving all installation variants to the top level)